source/multigrid/mg_level_global_transfer.cc

```diff
@@ -143,106 +143,143 @@ MGLevelGlobalTransfer<VectorType>::memory_consumption () const
 
 /* ------------------ MGLevelGlobalTransfer<VectorType> ----------------- */
 
+namespace
+{
+  template <int dim, int spacedim, typename Number>
+  void fill_internal(const DoFHandler<dim,spacedim> &mg_dof,
```

Comment:
this is a copy-paste which uses variables passed to this function. Git confuses it for a complete removal/addition.